src/server/cluster/cluster_family.cc

```diff
@@ -816,7 +816,7 @@ void ClusterFamily::DflyMigrateFullSyncCut(CmdArgList args, ConnectionContext* c
   std::lock_guard lck(migration_mu_);
   auto migration_it =
       std::find_if(incoming_migrations_jobs_.begin(), incoming_migrations_jobs_.end(),
-                   [sync_id](const auto& el) { return el->GetSyncId() == sync_id; });
+                   [sync_id = sync_id](const auto& el) { return el->GetSyncId() == sync_id; });
```

Comment:
why?

Reply:
clang-12 does not compile this when we capture variables that were defined via `auto [...] =`